### PR TITLE
[15.0][MIG] account_payment_sale

### PR DESCRIPTION
**Pull request by koenloodts**
Depends on:

- [x] #870 
- [x] #873

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]

File: account_payment_sale/tests/common.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Copyright 2018 Camptocamp
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class CommonTestCase(SavepointCase):
+class CommonTestCase(TransactionCase):
```
Comment (i-vyshnevska) [inline account_payment_sale/tests/common.py:7]: I would prefer to keep SavepointCase

**Review by i-vyshnevska — COMMENTED** [review]
LG same comment about squashing translation commits

File: account_payment_sale/tests/common.py (hunk 1 above)
Comment (simahawk) [reply account_payment_sale/tests/common.py:7]: SavepointCase is deprecated from 15 and TransactionCase has been made as SavepointCase (yeah.... it sucks :sob:)

File: account_payment_sale/tests/__init__.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+from . import common
```
Comment (simahawk) [inline account_payment_sale/tests/__init__.py:1]: never import common file here, is not needed

File: account_payment_sale/tests/common.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# Copyright 2018 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class CommonTestCase(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (simahawk) [inline account_payment_sale/tests/common.py:10]:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

File: test-requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+git+https://github.com/OCA/bank-payment@refs/pull/873/head#subdirectory=setup/account_payment_partner
+git+https://github.com/OCA/bank-payment@refs/pull/870/head#subdirectory=setup/account_payment_mode
```
Comment (simahawk) [inline test-requirements.txt:2]:
```suggestion
odoo-addon-account_payment_partner @ git+https://github.com/OCA/bank-payment@refs/pull/873/head#subdirectory=setup/account_payment_partner
odoo-addon-account_payment_mode @ git+https://github.com/OCA/bank-payment@refs/pull/870/head#subdirectory=setup/account_payment_mode
```

**Review by simahawk — CHANGES_REQUESTED** [review]
LG overall

**Comment by simahawk** [comment]
@koenloodts thanks for your work. Next time please list dependencies in the description (see how I changed it) :pray:

**Review by i-vyshnevska — APPROVED** [review]

**Review by misern2 — APPROVED** [review]
:+1:

**Review by brendapaniagua — APPROVED** [review]
Functional Review! LGTM!

**Comment by aisopuro** [comment]
@koenloodts it looks like the PRs this PR depends on have been merged. Do you have time to look through the review left by `simahawk`?

The runbot job is also failing, but I feel like I've seen that a lot lately, and I'm not sure how to fix it: possibly it has something to do with the switch to runboat?

File: test-requirements.txt (hunk 4 above)
Comment (yajo) [reply test-requirements.txt:2]: This can be removed as both PRs are now merged.

**Review by yajo — APPROVED** [review]

**Comment by rafaelbn** [comment]
Hello @simahawk !!! 😄 This PR is in you hands to be merged!

**Review by rafaelbn — APPROVED** [review]
👍

**Comment by simahawk** [comment]
the migration commit still contains the pending merges in test-requirements.
@koenloodts can you drop them?
Also, next time, pls remember to add them in a specific commit (eg: `[FIXME|DROPEME]: add TMP test dependencies`) :pray:

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Comment by ValentinVinagre** [comment]
@koenloodts do you have time to finish this pr?

**Comment by rafaelbn** [comment]
I have ping @koenloodts in several pull request from diferents repos and it looks he don't receive any email/notification because we doesn't answer at all 😢 . So...

**Comment by ValentinVinagre** [comment]
> I have ping @koenloodts in several pull request from diferents repos and it looks we don't receive any email/notification because we doesn't answer at all cry . So...

we finished it then

**Comment by rafaelbn** [comment]
PLease could a PSC close this PR in favour of https://github.com/OCA/bank-payment/pull/913 because this one didn't have answer in months. Thanks

**Pull request closed without merging**